spring-integration-webflux/src/test/java/org/springframework/integration/webflux/outbound/WebFluxRequestExecutingMessageHandlerTests.java

```diff
@@ -44,13 +46,15 @@
 import org.springframework.web.reactive.function.client.WebClient;
 import org.springframework.web.reactive.function.client.WebClientResponseException;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
```

Comment:
This is wrong imports re-order according our new code style.
Please, rebase to the latest `main` and see `src/idea` or `src/eclipse` for editor configs to import into your IDE.

Reply:
Import sequence has been updated to follow code style